build.gradle

```diff
@@ -228,29 +228,27 @@ task syncFrontendPublic(type: Sync) {
     }
 }
 
-task macHotReloadFrontend(type: Exec) {
-    dependsOn(installFrontend)
+task macHotReloadFrontend(type: com.liferay.gradle.plugins.node.tasks.ExecuteNpmTask) {
     onlyIf {os.isMacOsX()}
     workingDir 'frontend/'
-    commandLine 'npm', 'run', 'serveOpen'
+    args = ['run', 'serveOpen']
 }
 
 task windowsHotReloadFrontend(type: Exec) {
```

Comment:
Does this need to be updated to `com.liferay.gradle.plugins.node.tasks.ExecuteNpmTask` too?

Reply:
yes, which is why it is on hold.